src/main/java/it/pagopa/pm/gateway/service/CcResumeStep1Service.java

```diff
@@ -69,6 +75,8 @@ public void startResumeStep1(CreditCardResumeRequest request, String requestId)
             return;
         }
 
+        this.clientId = entity.getClientId();
```

Comment:
* What happens if `startResumeStep1` gets called twice before _any_ other callees get to execute (i.e. before `processResume`)?
If I'm not mistaken, the second call would overwrite `this.clientId` and both calls to `processResume` would process the request with the same clientId

* `this.clientId` is used only in the following call chain:
`startResumeStep1` > `processResume` > `executeStep1` > `executeAccount` > `executePatchTransaction`
As such, it can be safely passed as an argument in all of these functions

* On top of the last point, why not pass `clientId` directly?

Reply:
https://github.com/pagopa/pagopa-payment-transactions-gateway/pull/206/commits/406cf3b4aedf233f0651a6f2405ea0fb94ccf3db